deploy/roles/nginx/files/playreading.conf

```diff
@@ -36,15 +36,9 @@ server {
     quic_retry on;
     ssl_early_data on;
 
-    # Compress responses with gzip
-    gzip on;
-    gzip_vary on;
-    gzip_comp_level 9; # Max compression
```

Comment:
This gzip section is responsible for compressing responses on demand. This has been moved into the @nodesite section since we still want to compress responses from our api (which can't be compressed ahead of time).